.gitignore

```diff
@@ -17,3 +17,5 @@
 
 # Build
 build/
+docs/_build/
+CMakeCache.txt
```

Comment:
I don't think we need explicitly ignore CMakeCache.txt file if you adds whole build folder to ignore. 
BTW can we align build folder with the main one and also use just "build" without underscore?

Reply:
@mkrainiuk , Ok, let me consult with Todd R. He just showed me how to do a sphinx build from within the docs folder and we specified the _build directory for output of docs only. If there is a way to output to the main build directory, I will be happy to do that, but let me investigate first.

Reply:
Hi @mkrainiuk , Ok, after talking with Todd, my next commit will address both of your concerns here.